src/test/java/core/basesyntax/service/CsvFruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFruitTransactionParserImplTest {
+    private FruitTransactionParser parser = new CsvFruitTransactionParserImpl();
+
+    @Test
+    public void csvFruitTransactionParserImpl_correctData_Ok() {
```

Comment:
```suggestion
    public void parse_correctData_Ok() {
```
fix method names
you should use pattern `<method name>_<description>_<expected behaviour>`